File: morningSnacks/java/StrongNumber.java
import java.util.Scanner;

public class StrongNumber {
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int number = input.nextInt();

        int factorial = 1;
        int secondFactorial = 0;
        int newNumber = number;

        while (newNumber != 0) {
            int lastDigit = newNumber % 10;
            for (int count = 1; count <= lastDigit; count++) {
                factorial *= count;
            }
            newNumber = newNumber / 10;
            secondFactorial += factorial;
            factorial = 1;
        }

        if (number == secondFactorial) {
            System.out.println("Its a strong number");
        }
    }
}

File: morningSnacks/python/strongNumber.py
number = int(input("Enter a number: "))

factorial = 1
secondFactorial = 0
newNumber = number

while newNumber != 0:
    lastDigit = newNumber % 10
    for count in range(1, lastDigit + 1):
        factorial *= count
    newNumber = newNumber // 10
    secondFactorial += factorial
    factorial = 1

if number == secondFactorial:
    print("Its a strong number")

